repos/spack_repo/builtin/packages/acts/package.py

```diff
@@ -300,6 +300,12 @@ class Acts(CMakePackage, CudaPackage):
     variant("legacy", default=False, description="Build the Legacy package")
     variant("mlpack", default=False, description="Build MLpack plugin", when="@25:31")
     variant("onnx", default=False, description="Build ONNX plugin")
+    variant(
+        "torch",
+        default=True,
```

Comment:
Do we want this to be default=True? It seems different from the way other plugins are treated (default false), and it pulls in a much larger dependency too.

```suggestion
        default=False,
```